dapp_user/infrastructure/repositories/user_repository.py

```diff
@@ -46,3 +46,15 @@ def get_user_preferences(self, user_row_id):
     def delete_user(self, username):
         query = "DELETE FROM user WHERE username = %s "
```

Comment:
Why do we need Delete User?

Reply:
this is old flow. PR does not have any changes related to delete user